cmake/alpakaCommon.cmake

```diff
@@ -520,6 +523,138 @@ if(ALPAKA_ACC_GPU_HIP_ENABLE)
 
 endif() # HIP
 
+#-------------------------------------------------------------------------------
+# SYCL settings
+if(ALPAKA_ACC_SYCL_ENABLE)
+    # Possible SYCL platforms
+    cmake_dependent_option(ALPAKA_SYCL_PLATFORM_ONEAPI "Enable Intel oneAPI platform for the SYCL back-end" OFF "ALPAKA_ACC_SYCL_ENABLE" OFF)
+    cmake_dependent_option(ALPAKA_SYCL_PLATFORM_XILINX "Enable Xilinx platform for the SYCL back-end" OFF "ALPAKA_ACC_SYCL_ENABLE" OFF)
```

Comment:
Why does the user need to specify the SYCL platform? Can we not detect it?

Reply:
Unfortunately not. We support CMake 3.18 in which the Intel and Xilinx compilers are identified as a standard clang. In later CMake versions this changes to `IntelLLVM` but since Xilinx is an Intel fork we have no way to distinguish between the two.